{"name": "test_excel[0% no-repeat;background-size:100px 110px;background-image:url(https:\\/\\/ss1.bdstatic.com\\/5eN1bjq8AAUYm2zgoY3K\\/r\\/www\\/cache\\/s]", "status": "failed", "statusDetails": {"message": "AssertionError: assert b'0% no-repea...w\\\\/cache\\\\/s' == '<Response [200]>'\n  +b'0% no-repeat;background-size:100px 110px;background-image:url(https:\\\\/\\\\/ss1.bdstatic.com\\\\/5eN1bjq8AAUYm2zgoY3K\\\\/r\\\\/www\\\\/cache\\\\/s'\n  -'<Response [200]>'", "trace": "data = b'0% no-repeat;background-size:100px 110px;background-image:url(https:\\\\/\\\\/ss1.bdstatic.com\\\\/5eN1bjq8AAUYm2zgoY3K\\\\/r\\\\/www\\\\/cache\\\\/s'\n\n    @pytest.mark.parametrize('data', ExcelHandler.get_excel_data(1))\n    def test_excel(data):\n>       assert data == '<Response [200]>'\nE       AssertionError: assert b'0% no-repea...w\\\\/cache\\\\/s' == '<Response [200]>'\nE         +b'0% no-repeat;background-size:100px 110px;background-image:url(https:\\\\/\\\\/ss1.bdstatic.com\\\\/5eN1bjq8AAUYm2zgoY3K\\\\/r\\\\/www\\\\/cache\\\\/s'\nE         -'<Response [200]>'\n\ntest.py:16: AssertionError"}, "parameters": [{"name": "data", "value": "<class 'bytes'>"}], "start": 1586663324129, "stop": 1586663324130, "uuid": "5506bfc5-6726-4ac4-a15c-4ab093f93a74", "historyId": "f4d9ba4c698272a7fde5032f9e58d99b", "testCaseId": "1a84b5fa1cd8fc91bc1e7f314e541701", "fullName": "allure.test#test_excel", "labels": [{"name": "parentSuite", "value": "allure"}, {"name": "suite", "value": "test"}, {"name": "host", "value": "localhost"}, {"name": "thread", "value": "42329-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "allure.test"}]}